src/store/actions.js

```diff
@@ -184,6 +185,17 @@ export default {
 							resolve(title);
 						});
 					},
+					onEditSubjectLineAndPreviewText: ({ subjectLine, previewText }) => {
+						return new Promise((resolve) => {
+							resolve({ subjectLine, previewText });
```

Comment:
shouldn't the resolved value be empty?
Because in the sdk docs you wrote that it should resolve with nothing